main.py

```diff
@@ -27,101 +24,338 @@ async def play(ctx):
+                    name="Sobota",
+                    value="saturday"
+                ),
+                create_choice(
+                    name="Niedziela",
+                    value="sunday"
+                )
+            ]
+        )
+    ]
+
+)
+async def _schedule(ctx: SlashContext, day: str):
+    if ctx.guild is None:
+        await ctx.send("Komenda nie działa w wiadomości prywatnej!")
+        return
+    resp = requests.get("https://radio17.pl/api/schedule")
+    resp_json = resp.json()
+    pages = []
+    if day.lower() == "short":
+        short_embed = discord.Embed(title="Dziś jeszcze gramy:")
+        for broadcast in resp_json[week_days[datetime.datetime.today().weekday()]]:
+            if int(broadcast["end_hour"][:2]) > datetime.datetime.today().hour:
+                short_embed.add_field(name=broadcast["title"],
+                                      value=broadcast["start_hour"] + " - " + broadcast["end_hour"], inline=False)
+        await ctx.send(embed=short_embed)
+        return
+    if day.lower() not in week_days:
+        await ctx.send("Wpisz poprawny dzień tygodnia w języku angielskim!")
+        return
+    for broadcast in resp_json[day]:
+        embed = discord.Embed(title=broadcast["title"])
+        embed.add_field(name="Nazwa audycji:", value=broadcast["title"], inline=False)
+        embed.add_field(name="Od - do:", value=broadcast["start_hour"] + " - " + broadcast["end_hour"], inline=False)
+        if broadcast["description"] is not None:
+            embed.add_field(name="Opis:", value=broadcast["description"], inline=False)
+        embed.set_image(url=broadcast["player_image"])
+        pages.append(embed)
+    message = await ctx.send(embed=pages[0])
+    await message.add_reaction('⏮')
+    await message.add_reaction('◀')
+    await message.add_reaction('▶')
+    await message.add_reaction('⏭')
+
+    def check(reaction, user):
+        return user == ctx.author
+
+    i = 0
+    reaction = None
+    while True:
+        if str(reaction) == '⏮':
+            i = 0
+            await message.edit(embed=pages[i])
+        elif str(reaction) == '◀':
+            if i > 0:
+                i -= 1
+                await message.edit(embed=pages[i])
+        elif str(reaction) == '▶':
+            if i < len(pages) - 1:
+                i += 1
+                await message.edit(embed=pages[i])
+        elif str(reaction) == '⏭':
+            i = len(pages) - 1
+            await message.edit(embed=pages[i])
+        try:
+            reaction, user = await bot.wait_for('reaction_add', timeout=30.0, check=check)
+            await message.remove_reaction(reaction, user)
+        except:
+            break
+    await message.clear_reactions()
+
+
+@bot.event
+async def on_command_error(ctx, error):
+    if isinstance(error, commands.errors.CommandOnCooldown):
+        await ctx.channel.send('This command is on a %.2fs cooldown' %
+                               error.retry_after)
```

Comment:
```suggestion
        await ctx.channel.send(f'This command is on a {error.retry_after:.2f}s cooldown')
```

Chyba tak, albo %.2f, nie pamiętam bez interpretera przez oczami